app/templates/buyers/index.html

```diff
@@ -0,0 +1,60 @@
+{% extends "_base_page.html" %}
+{% block page_title %}Your account - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+    {
+        "link": "/",
+        "label": "Digital Marketplace"
+    }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+<div class="grid-row">
+  <div class="column-full">
+    {% with
+        context = current_user.email_address,
+        heading = current_user.name
+    %}
+    {% include 'toolkit/page-heading.html' %}
+    {% endwith %}
+  </div>
+  <div class="column-full dm-account-overview-options">
+    
+    <div class="framework">
+      <div class="dmspeak">
+        <h3 class="heading-xmedium">Cloud hosting, software and support</h3>
+      </div>
+      <p>Buy cloud technology through the G-Cloud framework.</p>
+      <p>
+        {% if user_projects_total %}
+          <a href="/buyers/direct-award">View your saved searches</a><br>
+        {% else %}
+          You don't have any saved searches
+        {% endif %}
+      </p>
+    </div>
+
+    <div class="framework">
+      <div class="dmspeak">
+        <h3 class="heading-xmedium">Digital outcomes, specialists and user research</h3>
+      </div>
+      <p>Buy services through the Digital Outcomes and Specialists framework.</p>
```

Comment:
Same here - the briefs pages don't mention "framework" anywhere, we really shouldn't be introducing it into content at all.